openverse_catalog/dags/common/loader/loader.py

```diff
@@ -40,10 +40,13 @@ def load_from_s3(
     media_type,
     tsv_version,
     identifier,
+    ti,
 ):
     sql.load_s3_data_to_intermediate_table(
         postgres_conn_id, bucket, key, identifier, media_type
     )
-    sql.upsert_records_to_db_table(
+    record_count = sql.upsert_records_to_db_table(
         postgres_conn_id, identifier, media_type=media_type, tsv_version=tsv_version
     )
+
+    ti.xcom_push(key=f"{media_type}_record_count", value=record_count)
```

Comment:
I think we can remove the need to use `ti` in this function by just returning `record_count`! By default, the `PythonOperator` pushes the return value of a called function to the XComs already (under the `return_value` key, which is the default when calling `ti.xcom_pull` I believe).

More on that here: https://airflow.apache.org/docs/apache-airflow/stable/concepts/xcoms.html#xcoms

It's not the most pressing change, we could definitely leave as-is. But it prevents us from needing to use Airflow-specific resources in this function at least. Up to you!

Reply:
Whoa, slick! That's great to know, I'll definitely update.